### PR TITLE
Fix libressl use

### PR DESCRIPTION
**Pull request by SoapGentoo**

### CODE REVIEW

File: net-libs/libssh2/libssh2-1.8.0-r2.ebuild [hunk 1]
```diff
@@ -12,16 +12,18 @@ LICENSE="BSD"
...
-"
+REQUIRED_USE="?? ( gcrypt mbedtls )"
 RESTRICT="test"
 
 RDEPEND="
-	!libressl? ( >=dev-libs/openssl-1.0.1h-r2:0=[${MULTILIB_USEDEP}] )
 	gcrypt? ( >=dev-libs/libgcrypt-1.5.3:0[${MULTILIB_USEDEP}] )
-	libressl? ( dev-libs/libressl:0=[${MULTILIB_USEDEP}] )
-	mbedtls? ( net-libs/mbedtls[${MULTILIB_USEDEP}] )
+	!gcrypt? (
```
[the remark this answers is not in the document]
Comment (SoapGentoo) [reply net-libs/libssh2/libssh2-1.8.0-r2.ebuild]: The question is, how do you only pull in openssl/libressl if `USE="-gcrypt -mbedtls"`?

**Comment by hsoft** [comment]
Regarding mupdf: I'll have to revisit it, but if I remember correctly, mupdf always needs SSL, so the "ssl && libressl" semantic doesn't work.

File: app-text/mupdf/mupdf-1.14.0-r3.ebuild [hunk 2]
```diff
@@ -78,15 +79,11 @@ src_prepare() {
 
 _emake() {
 	# When HAVE_OBJCOPY is yes, we end up with a lot of QA warnings.
-	local have_libcrypto=no
```
Comment (SoapGentoo) [inline app-text/mupdf/mupdf-1.14.0-r3.ebuild]: @hsoft why does it have this option then?

Comment (hsoft) [reply app-text/mupdf/mupdf-1.14.0-r3.ebuild]: Hum, yeah. it seems that I don't remember correctly... Should be fine then.

File: net-libs/libssh2/libssh2-1.8.0-r2.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (mgorny) [reply net-libs/libssh2/libssh2-1.8.0-r2.ebuild]: Well, you could do `!gcrypt? ( !mbedtls? ( ... ) )` but I guess this way is more readable.

**Pull request closed without merging**